client/src/App.js

```diff
@@ -1,11 +1,69 @@
+import AddVideo from "./componets/ADDVIDEO/AddVideo";
+import SingleVideo from "./componets/SINGLEVIDEO/SingelVideo";
+import Search from "./componets/Search/Search";
+import Data from "./VideoData.json";
```

Comment:
You've stored the data from VideoData.json in a variable called `Data` - there is a slight issue with this variable name. Can you think what it is?

Reply:
@Dedekind561 thank you for the feedback. Yes i have already commented that part as i am using the api now

Reply:
@Dedekind561 i also have changed the colors for the like and dislike